### PR TITLE
Bump to version 3.0.0

### PR DESCRIPTION
**Pull request by fitzgen**
@TethysSvensson want to give this a once over before I merge and publish? Thanks!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,149 @@
...
+
+### Changed
+
+* **BREAKING:** The direction we allocate within a chunk has changed. It used to
+  be "upwards", from low addresses within a chunk towards high addresses. It is
+  now "downwards", from high addresses towards lower addresses.
+
+  Additionally, the order in which we iterate over allocated chunks has changed!
+  We used to iterate over chunks from oldest chunk to newest chunk, and now we
+  do the opposite: the toungest chunks are iterated over first, and the oldest
```
Comment (TethysSvensson) [inline CHANGELOG.md]: youngest?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,149 @@
...
+  chunks are iterated over last.
+
+  If you were using `Bump::each_allocated_chunk` to iterate over data that you
+  had previously allocated, and *you want to iterate in order of allocation*,
+  you need to reverse the chunks iterator and also reverse the order in which
+  you loop through the data within a chunk!
```
Comment (TethysSvensson) [inline CHANGELOG.md]: Have you considered also writing something about the maximum supported alignment size? I am not sure if it actually matters enough though.

**Comment by TethysSvensson** [comment]
This looks good to me. I cannot think if anything else you need to do before a release. I cannot remember anything you missed, nor could I find something in a brief look through `git log` or `git diff`.

**Comment by fitzgen** [comment]
Thanks for taking a look! I'll merge once CI is green again, and then publish.

**Comment by fitzgen** [comment]
Published!

**Pull request merged**